azure-webapp-maven-plugin/README.md

```diff
@@ -1,186 +1,179 @@
 # Maven Plugin for Azure App Service
+
+## Table of Content
+  - [Overview](#overview)
+  - [Prerequisites](#prerequisites)
+  - [Quick Start](#quick-start)
+  - [Goals](#goals)
+      - [`azure-webapp:config`](#azure-webappconfig)
+      - [`azure-webapp:deploy`](#azure-webappdeploy)
+  - [Configuration Details](#configuration-details)
+      - [`<javaVersion>`](#javaversion)
+      - [`<webContainer>`](#webcontainer)
+      - [`<resource>`](#resource)
+      - [`<region>`](#region)
+      - [`<pricingTier>`](#pricingtier)
+  - [Samples](#samples)
+  - [Feedback and Questions](#feedback-and-questions)
+  - [Data and Telemetry](#data-and-telemetry)
+
+## Overview
 [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-webapp-maven-plugin.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22azure-webapp-maven-plugin%22)
 
-The Maven Plugin for Azure App Service provides seamless integration into Maven projects, 
-and makes it easier for developers to deploy to different kinds of Azure Web Apps:
+The Maven Plugin for Azure App Service provides seamless integration into Maven projects,
+and makes it easier for developers to deploy to different kinds of [Azure Web Apps](https://docs.microsoft.com/en-us/azure/app-service/):
   - [Web App on Linux](https://docs.microsoft.com/azure/app-service-web/app-service-linux-intro)
   - [Web App on Windows](https://docs.microsoft.com/en-us/azure/app-service/app-service-web-overview)
   - [Web App for Containers](https://docs.microsoft.com/en-us/azure/app-service/containers/tutorial-custom-docker-image)
-  
-#### Table of Content
-- [Prerequisites](#prerequisites)
-- [Quick Start](#quick-start)
-- [Samples](#samples)
-- [Goals](#goals)
-- [Authentication with Azure](#authentication-with-azure)
-- [Configurations](#configurations)
+
 
 ## Prerequisites
 
 Tool | Required Version
 ---|---
 JDK | 1.7 or above
 Maven | 3.0 or above
-## Quick Start 
-
-1. Create a new Azure App Service and choose Linux with built-in tomcat as the environment. 
-
-2. To use this plugin in your Maven project, add the following settings for the plugin to your `pom.xml` file:
+## Quick Start
+1. Make sure you have already authenticated in Azure. You can use the Azure CLI 2.0 for authentication. More authentication methods can be found [here](../docs/common-configuration.md).
 
+   - Install the Azure CLI 2.0 by following the instructions in the [Install Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) article.
+   - Run the following commands to log into your Azure subscription：
+    ```
+        $ az login
+        $ az account set --subscription <subscription Id>
+    ```
+2. You could create a new Azure App Service and fill in the information in the `pom.xml`. This configuration can also be generated by [`azure-webapp:config`](#azure-webappconfig)
```

Comment:
Miss period. 
This configuration can also be generated by [`azure-webapp:config`](#azure-webappconfig)
-> 
This configuration can also be generated by [`azure-webapp:config`](#azure-webappconfig).